src/classic/types/__tests__/ReactPropTypes-test.js

```diff
@@ -520,6 +520,12 @@ describe('ReactPropTypes', function() {
   });
 
   describe('OneOf Types', function() {
+  	it("should fail for invalid argument", function() {
+  		var error = PropTypes.oneOf('red', 'blue');
```

Comment:
We don't have lint properly running in test files yet, but can you make sure these are converted to 2 spaces as well